src/App.tsx

```diff
@@ -1,10 +1,52 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState, useEffect, useMemo } from 'react';
 import React from 'react';
+import { ErrorMessage } from './types/ErrorMessage';
 import { UserWarning } from './UserWarning';
+import {getTodos} from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Todo } from './types/Todo';
+import { ToDoList } from './components/ToDoList/ToDoList';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorNotification } from './components/ErrorMessage/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 7035;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [ todoStatus, setTodoStatus ] = useState(TodoStatus.ALL);
+  const [errorMessage, setErrorMessage]
+  = useState<ErrorMessage>(ErrorMessage.NONE);
```

Comment:
```suggestion
  const [errorMessage, setErrorMessage] = useState(ErrorMessage.NONE);
```

Reply:
fit this comment
https://github.com/mate-academy/react_todo-app-loading-todos/pull/453/files#r1173693186